tests/Infrastructure/Aggregate/PreconditionsTree/BeginnerPreconditionsUnitTest.php

```diff
@@ -52,13 +52,13 @@ public function testFulfilled(): void
         $stagingDir = $this->stagingDir->reveal();
         $exclusions = $this->exclusions;
         $this->commonPreconditions
-            ->assertIsFulfilled($activeDir, $stagingDir, $exclusions)
+            ->assertIsFulfilled($activeDir, $stagingDir, $exclusions, $exclusions)
```

Comment:
ignore all these, phpstorm refactoring helpfully just dupes the argument if it has no idea.
if there's broad support for the idea I'll fix all these